firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/metadata/MetaDataStore.java

```diff
@@ -231,8 +228,9 @@ private static String keysDataToJson(final Map<String, String> keyData) {
   }
 
   private static List<RolloutAssignment> jsonToRolloutsState(String json) throws JSONException {
+    final JSONObject object = new JSONObject(json);
+    final JSONArray dataArray = object.getJSONArray("rolloutsState");
```

Comment:
Could we go with `rollouts_state` to stay consistent with proto style for underscores?

Reply:
I think how encodable works on Android, everything is in camel case. I have checked other models, they are all camel cased

Reply:
Discussed offline